datatypes/float.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2018 gopcua authors. All rights reserved.
+// Use of this source code is governed by a MIT-style license that can be
+// found in the LICENSE file.
+
+package datatypes
+
+import (
+	"encoding/binary"
+	"math"
+)
+
+// All floating-point values shall be encoded with the appropriate IEEE-754 binary representation
```

Comment:
Could you add a line starting from Float like other structs do?
Otherwise this comments won’t be parsed as GoDoc correctly.